Example/DocumentScannerApp/Bundle+UserActivityIdentifier.swift

```diff
@@ -10,7 +10,10 @@ import Foundation
 
 extension Bundle {
     var userActivityIdentifier: String {
-        let nsUserActivityTypes = Bundle.main.object(forInfoDictionaryKey: "NSUserActivityTypes")! as! [String]
-        return nsUserActivityTypes.first!
+        guard let nsUserActivityTypes = object(forInfoDictionaryKey: "NSUserActivityTypes") as? [String],
+            let activityId = nsUserActivityTypes.first else {
+            return "error"
```

Comment:
In this case I would use a `fatalError()`, to indicate a programmer error.

If the project setup is inconsistent, there's nothing that the app can do to recover, so we might as well crash.

@fbernutz what do you think?

Reply:
Yes, I was thinking the same. But please add a message to fatalError, for example: `fatalError("Need to declare at least one NSUserActivityTypes")`

Reply:
Fixed. :)